src/Uno.UI/UI/Xaml/Controls/CalendarView/Primitives/CalendarPanel.ModernCollectionBasePanel.cs

```diff
@@ -432,7 +432,15 @@ internal void ScrollItemIntoView(int index, ScrollIntoViewAlignment alignment, d
 				// then it will request GetContainerFromIndex and tries to focus it.
 				// So here we prepare the _effectiveViewport (which will most probably be re-updated by the ChangeView below),
 				// and then force a base_Measure()
-				_effectiveViewport.Y += newOffset - currentOffset;
+
+				if (newOffset >= sv.ScrollableHeight)
```

Comment:
If this is not coming from WinUI sources, then it should be wrapped in `#if HAS_UNO` with a `//` explanation of the issue

Reply:
done!